client/components/review.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+export default class Review extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      theButtonClass: '',
+      theButtonText: ''
+    };
+  }
+
+  componentDidMount() {
+    if (this.props.review.rating === 'inform') {
+      this.setState({ theButtonClass: 'inform no-button me-4 mt-2 text-dark', theButtonText: 'Informative' });
+    } else {
+      this.setState({ theButtonClass: 'persuade no-button me-4 mt-2', theButtonText: 'Persuasive' });
+    }
+  }
```

Comment:
In general we want to avoid storing things like this in state. instead compute these values as variables in your render method.